### PR TITLE
ADD: new custom install layouts

### PR DESCRIPTION
**Pull request by MaxTheGeeek**

### CODE REVIEW

**Review by montajebi — APPROVED** [review]

**Review by NeoPlays — CHANGES_REQUESTED** [review]
lgtm otherwise

File: launcher/src/components/UI/TheCarousel/components/TheCarousel.vue [hunk 1]
```diff
@@ -203,23 +209,38 @@ const linkPicker = async (item) => {
...
-      break;
-    case 3:
-      selectedLinks.value = installStore.sepolia;
-      break;
-    case 4:
-      selectedLinks.value = installStore.gnosis;
-      break;
-    default:
-      break;
+  if (configNetwork.value.id === 1 || configNetwork.value.id === 1) {
```
[the remark this answers is not in the document]
Comment (MaxTheGeeek) [reply launcher/src/components/UI/TheCarousel/components/TheCarousel.vue:212]: let me check

File: launcher/src/components/UI/TheCarousel/components/TheCarousel.vue [hunk 2]
```diff
@@ -209,7 +209,9 @@ const linkPicker = async (item) => {
 };
 
 const setSelectedLinks = () => {
-  if (configNetwork.value.id === 1 || configNetwork.value.id === 1) {
+
+  if (configNetwork.value.id === 1 || currentNetwork.value.id === 1) {
+
     selectedLinks.value = installStore.mainnet;
   } else if (configNetwork.value.id === 2 || configNetwork.value.id === 2) {
```
Comment (NeoPlays) [inline launcher/src/components/UI/TheCarousel/components/TheCarousel.vue:216]: this goes for all `ifs` tho :sweat_smile:

**Review by NeoPlays — CHANGES_REQUESTED** [review]

**Review by NeoPlays — APPROVED** [review]
lgtm

**Pull request merged**